weblate/static/editor/full.js

```diff
@@ -324,6 +324,26 @@
         error: function (jqXHR, textStatus, errorThrown) {
           addAlert(errorThrown);
         },
+        success: function (data) {
+          if (dismiss_all) {
+            var $flagBox = $('[data-focus="#id_extra_flags"]').closest(
+              "div.list-group-item"
+            );
+            if ($flagBox.children("em").length) {
+              $flagBox.find("em").remove();
+              $flagBox.append(data);
+            } else {
+              var flagsTextNode = $flagBox.contents().filter(function () {
+                return (
+                  this.nodeType === 3 &&
+                  (this.nextElementSibling === null ||
+                    this.nextElementSibling.nodeName.toLowerCase() !== "h5")
+                );
+              })[0];
+              flagsTextNode.textContent = data;
+            }
+          }
+        },
```

Comment:
Instead of the template should be adjusted to contain single element to contain this:

https://github.com/WeblateOrg/weblate/blob/f6753a1a1c63fade6ad418fbda827c6750ab0bda/weblate/templates/translate.html#L648-L652

Reply:
I wrapped this section in a div with an id that can be easily targeted by javascript